src/Wizard/Wizard.less

```diff
@@ -20,6 +20,10 @@
   }
 }
 
+.Wizard--stepsDisplay--stepTitle {
+  -ms-flex: 0 1 auto;
+}
+
```

Comment:
linter caught that this `.Wizard--stepsDisplay--stepTitle` selector is already there on line 115, so just add it there?